svgnative/src/SVGStringParser.cpp

```diff
@@ -712,7 +730,7 @@ void ParsePathString(const std::string& pathString, Path& p)
                 currentY += newY;
             }
 
-            SVG_ASSERT(angle == 0);
+            //SVG_ASSERT(angle == 0);
```

Comment:
We should remove it entirely.

Reply:
Are you sure, I filed an issue #134 that shows problems with arcs, so I think the assertion is there for a good reason. I commented it out by mistake, I'm removing the comment and putting the assertion back for now.

Reply:
Sounds good then.